view/admin/components/common/Dropdown.js

```diff
@@ -179,7 +179,7 @@ class Dropdown extends React.Component {
     let outputUL = '';
     // if options exist, load them
     if (data.length) {
-      outputUL = <ul className="slds-dropdown__list" role="menu">
+      outputUL = <ul className="slds-lookup__list" role="menu">
```

Comment:
It looks like the "New Perspective" label is not aligned with the names of the perspectives--it looks a few pixels to the right.